proto/multistaking/v1/query.proto

```diff
@@ -34,6 +34,9 @@ service Query {
     option (google.api.http).get =
         "/realiotech/multistaking/v1/validator-coin/{validator_addr}";
   }
+  rpc BondWeights(QueryBondWeightsRequest) returns (QueryBondWeightsResponse) {
+    option (google.api.http).get = "/realiotech/multistaking/v1/weights";
```

Comment:
We need to [RegisterGRPCGatewayRoutes](https://github.com/realiotech/multi-staking/blob/346ccbb23f1ab3d07f65292b5539590be1ecd7c6/x/multi-staking/module.go#L67) if we want to allow REST API